components/stepper/overview.md

```diff
@@ -16,25 +16,16 @@ The <a href ="https://www.telerik.com/blazor-ui/stepper"><strong>Stepper for Bla
 The Stepper provides several features out of the box, including the ability to define the appearance of each [step]({%slug stepper-steps-overview%}), [validation]({%slug stepper-steps-validation%}) for the steps, strict [linear flow]({%slug stepper-linear-flow%}) and [step template]({%slug step-template%}) to make the it fit any design requirements.
 
 
-#### In this article:
-   * [Basics](#basics)
-   * [Features](#features)
+## Creating Blazor Stepper
```

Comment:
The code snippet should reflect the instructions before that. Currently, the instructions do not mention anything about `Value`, `Label` and `Icon`.